Traceback (most recent call last):
  File "/home/john/anaconda3/envs/appstore/lib/python3.10/site-packages/jupyter_cache/executors/utils.py", line 58, in single_nb_execution
    executenb(
  File "/home/john/anaconda3/envs/appstore/lib/python3.10/site-packages/nbclient/client.py", line 1305, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
  File "/home/john/anaconda3/envs/appstore/lib/python3.10/site-packages/jupyter_core/utils/__init__.py", line 166, in wrapped
    return loop.run_until_complete(inner)
  File "/home/john/anaconda3/envs/appstore/lib/python3.10/asyncio/base_events.py", line 649, in run_until_complete
    return future.result()
  File "/home/john/anaconda3/envs/appstore/lib/python3.10/site-packages/nbclient/client.py", line 705, in async_execute
    await self.async_execute_cell(
  File "/home/john/anaconda3/envs/appstore/lib/python3.10/site-packages/nbclient/client.py", line 1058, in async_execute_cell
    await self._check_raise_for_error(cell, cell_index, exec_reply)
  File "/home/john/anaconda3/envs/appstore/lib/python3.10/site-packages/nbclient/client.py", line 914, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
with pd.option_context('format.precision',2):
    df1_style = freq.style.set_table_attributes("style='display:inline; margin-right:220px;'").set_caption("Average User Rating Frequency Distribution")
    df2_style = desc.style.set_table_attributes("style='display:inline; margin-right:120px'").set_caption("Average User Rating Descriptive Statistics")
    display_html(df1_style._repr_html_() + df2_style._repr_html_(), raw=True)
------------------


[0;31m---------------------------------------------------------------------------[0m
[0;31mNameError[0m                                 Traceback (most recent call last)
Cell [0;32mIn[14], line 2[0m
[1;32m      1[0m [38;5;28;01mwith[39;00m pd[38;5;241m.[39moption_context([38;5;124m'[39m[38;5;124mformat.precision[39m[38;5;124m'[39m,[38;5;241m2[39m):
[0;32m----> 2[0m     df1_style [38;5;241m=[39m [43mfreq[49m[38;5;241m.[39mstyle[38;5;241m.[39mset_table_attributes([38;5;124m"[39m[38;5;124mstyle=[39m[38;5;124m'[39m[38;5;124mdisplay:inline; margin-right:220px;[39m[38;5;124m'[39m[38;5;124m"[39m)[38;5;241m.[39mset_caption([38;5;124m"[39m[38;5;124mAverage User Rating Frequency Distribution[39m[38;5;124m"[39m)
[1;32m      3[0m     df2_style [38;5;241m=[39m desc[38;5;241m.[39mstyle[38;5;241m.[39mset_table_attributes([38;5;124m"[39m[38;5;124mstyle=[39m[38;5;124m'[39m[38;5;124mdisplay:inline; margin-right:120px[39m[38;5;124m'[39m[38;5;124m"[39m)[38;5;241m.[39mset_caption([38;5;124m"[39m[38;5;124mAverage User Rating Descriptive Statistics[39m[38;5;124m"[39m)
[1;32m      4[0m     display_html(df1_style[38;5;241m.[39m_repr_html_() [38;5;241m+[39m df2_style[38;5;241m.[39m_repr_html_(), raw[38;5;241m=[39m[38;5;28;01mTrue[39;00m)

[0;31mNameError[0m: name 'freq' is not defined

